### ISSUE TITLE
[monitoring-satellite] we had to restart prometheus-k8s for gen49 to report normalized load average

### ISSUE THREAD

**Issue by kylos101** [gitpod-io/observability#160]
### Bug description

we had to restart prometheus-k8s for gen49 to report normalized load average

### Steps to reproduce

I believe the issue still exists (@aledbf shared it does yesterday), so, when we deploy gen50, we expect to have this problem again.

### Expected behavior

We should not have to restart prometheus-k8s

### Example repository

n/a

### Anything else?

[Internal Slack context](https://gitpod.slack.com/archives/C01KGM9EBD4/p1655947575965019)

**Comment by mads-hartmann** [comment]
@kylos101 Was this needed for Gen51 and Gen52 too? ☺️

**Comment by kylos101** [comment]
I thought we added a hack to "prevent this", but I am unable to find it. @aledbf does this ring any bells, or is this a non-issue now? I ask because you built the last few generations of clusters, and am not sure why we didn't have this issue.

**Comment by mads-hartmann** [comment]
Unassigning myself and moving back to scheduled as I'm heading on vacation. @kylos101 @aledbf would be great to know if this issue still happens though ☺️

**Comment by kylos101** [comment]
Hi @mads-hartmann , I think we're okay, and am going to close this, and am guessing an upstream change fixed this.

That or we just got lucky.

We did have to make [this change](https://github.com/gitpod-io/ops/commit/abfca2bf47a906ef8e6ced08c81e0d5cd5f614bf), but I'm not aware of any other bandaids.

cc: @aledbf @sagor999 are we deleting the prometheus pod, or restarting prometheus after a new cluster is deployed, or when we enable monitoring? If yes, let's reopen, I could not find evidence of either as a band aid.
